backend/src/main/java/ufrn/imd/boraPagar/user/UserController.java

```diff
@@ -12,10 +12,16 @@
 
 import ufrn.imd.boraPagar.core.AbstractController;
 import ufrn.imd.boraPagar.core.Views;
+import org.springframework.web.bind.annotation.PostMapping;
 
 @RestController
 @RequestMapping("users")
 public class UserController extends AbstractController<UserModel, UserService>{
+    @PostMapping("/welcome")
```

Comment:
Nesse caso a rota será users/welcome, certo? Não é meio estranho ter o users no plural sendo que é o login de uma só pessoa?

Ou tiramos o `RequestMapping("users")` do controller geral ou criamos outro controller. Fazendo o connections coloquei um UserController e renomiei esse para UsersController, acho que os nomes são muito similares, mas gosto da ideia principal.

Reply:
Faz sentido. Vou trabalhar nisso ainda hoje.

Reply:
@Kywal, você fala criar um controller só para o welcome?

Reply:
Não necessariamente, poderia ser do welcome e outras rotas, mas a ideia principal é por aí, modularizar os controllers.

Reply:
Agora entendi melhor tua ideia. Olha, geralmente é assim que acontece. Essa é uma discussão parecida com: "No banco de dados eu coloco User ou Users?", não tem um padrão específico pois vai depender do projeto. Com relação às nossas rotas, optamos por definir no plural mesmo `subjects` tudo que for de requisição relacionada a disciplinas. Isso não quer dizer necessariamente que tudo que vamos fazer nessa rota envolve diversas disciplinas pois desde o início podemos fazer `subjects?code=DIM0101` ou `subjects/12031209401293`.

Reply:
Partindo de `subjects?code=DIM0101`, faz sentido (ainda mais que seja `subjects/interesteds` ao invés de `subjects/interested` que `subjects/interesteds?code=DIM0101` também segue o mesmo padrão.

Reply:
Entendi. Pessoalmente eu optaria por fazer a rota do welcome puramente "/welcome", mas isso são só detalhes, o importante é modularizar.